builder/solver_monitor.go

```diff
@@ -194,8 +194,6 @@ Loop:
 				}
 
 			}
-		case <-ctx.Done():
```

Comment:
is this intentional?

Reply:
Yes, the status stream also ends when ctx is done, however there may be a few more important log messages that need to be printed before that.